src/test/java/com/google/javascript/clutz/DeclarationSyntaxTest.java

```diff
@@ -29,31 +29,41 @@
       new FilenameFilter() {
         @Override
         public boolean accept(File dir, String name) {
-          return TS_SOURCES.accept(dir, name) && !name.contains("_emit_platform_externs");
+          return TS_SOURCES.accept(dir, name) && !name.contains("_with_platform");
```

Comment:
This typo means "TS_SOURCES_WITHOUT_PLATFORM_EXTERNS" was taking all sources and "TS_SOURCES_WITH_PLATFORM_EXTERNS" was empty right?

Why not then just flatten everything into one list "TS_SOURCES", and clean up our code.

Reply:
No reason.  Done.